pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -131,6 +146,37 @@ func TestCreateFilesystem(t *testing.T) {
 	assert.ElementsMatch(t, []string{"rook-ceph-mds-myfs-a", "rook-ceph-mds-myfs-b"}, testopk8s.DeploymentNamesUpdated(deploymentsUpdated))
 	testopk8s.ClearDeploymentsUpdated(deploymentsUpdated)
 
+	// Increasing the number of data pools should be successful.
+	createdNewDataPool := false
+	associatedNewDataPool := false
+	executor = &exectest.MockExecutor{
+		MockExecuteCommandWithOutputFile: func(command string, outFileArg string, args ...string) (string, error) {
+			if contains(args, "fs") && contains(args, "get") {
+				return createdFs, nil
+			} else if equals(args[0:4], []string{"osd", "pool", "create", "myfs-data1"}) {
+				createdNewDataPool = true
+				return "", nil
+			} else if equals(args[0:4], []string{"fs", "add_data_pool", "myfs", "myfs-data1"}) {
+				associatedNewDataPool = true
+				return "", nil
+			}
+			return "{\"key\":\"mysecurekey\"}", nil
```

Comment:
Ditto the optional suggestion above about only returning this for a specific commnad.

Reply:
Done